File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/ParticelleCatastaliMacrouso.ts
import { Macrouso } from '../metaschema/Macrouso';
import { IntervalloTemporale } from './IntervalloTemporale';

export class ParticelleCatastaliMacrouso {
    macrouso: Macrouso;
    validita: IntervalloTemporale;
    Area: number;

    Piva: string;
    NumeroFascicolo: string;
    DataValidazioneFascicolo: Date;

}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/ParticelleCatastaliMacrouso.cs
using AgronicaCoreModelsSTD.metaschema;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class ParticelleCatastaliMacrouso
    {
        public Macrouso macrouso { get; set; }
        public IntervalloTemporale validita { get; set; }
        public double Area { get; set; }

        public string Piva { get; set; }
        public string NumeroFascicolo { get; set; }
        public DateTime DataValidazioneFascicolo { get; set; }


        public ParticelleCatastaliMacrouso()
        {

        }
    }
}
